PhysicsTools/PatAlgos/plugins/PATTauProducer.cc

```diff
@@ -386,6 +477,7 @@ void PATTauProducer::produce(edm::Event & iEvent, const edm::EventSetup & iSetup
 	// aTauPFEssential.svCov_ = tauLifetimeInfo.secondaryVertexCov();
       }
     }
+*/
```

Comment:
I had missed to remove this commented old code before committing. It's now cleaned up.